scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1359,10 +1376,13 @@ class Router(formatOps: FormatOps) {
             case x => throw new UnexpectedTree[Term.If](x)
           }) =>
         val expire = leftOwner.asInstanceOf[Term.If].elsep.tokens.last
+        val noSpace = style.newlines.source match {
```

Comment:
3-times repeated `noSpace` computation? May be extract it to a function?

Reply:
done!